internal/pkg/webhooks/binding/binding.go

```diff
@@ -81,6 +81,7 @@ func newContainerMap(spec *corev1.PodSpec) containerMap {
 // Leader election
 // nolint:lll
 // +kubebuilder:rbac:groups=core,namespace="security-profiles-operator",resources=configmaps;events,verbs=get;list;watch;create;update;patch
```

Comment:
Hm, the webhooks stills complains that it cannot create events in other namespaces, like `default`. Maybe we have to make this one more general?

Reply:
We could simply add
```
 // +kubebuilder:rbac:groups=core,resources=events,verbs=get;list;watch;create;update;patch
```

And that would allow the operator to issue events in all namespaces.

Reply:
Alright, I also removed the `events` from the resources here.